MainApp/templates/MainApp/base_homepage.html

```diff
@@ -252,17 +252,40 @@
 .like {
   border: none;
   border-radius: 80px;
-  padding: 12px 16px;
+  padding: 8px 8px;
   cursor: pointer;
   font-size: 16px;
   color: ghostwhite;
+  border-color: rgb(29,161,242,0.50);
+  border-style: solid;
+  border-radius: 32px;
   background-color: rgba(0,0,0,0);
+  color: rgb(29,161,242,2.00);
   outline: none
 }
 
 .like:hover {
-  color: rgb(138, 5, 5);
-  background-color: rgba(112, 3, 48, 0.15);
+  background-color: rgba(29, 160, 242, 0.15);
+}
+
+.unlike {
+  border: none;
+  border-radius: 80px;
+  padding: 8px 8px;
+  cursor: pointer;
+  font-size: 16px;
+  color: ghostwhite;
+  border-color: rgba(138, 5, 5, 0.747);
+  border-style: solid;
+  border-radius: 32px;
+  background-color: rgba(0,0,0,0);
+  color: rgba(138, 5, 5, 0.747);
+  outline: none
+}
+
+.unlike:hover {
+  background-color: rgba(179, 10, 80, 0.15);
+  color: rgba(138, 5, 5, 0.747);
```

Comment:
Refreshed the colors of the like/unlike buttons...it does not look the greatest right now but its a start. We can make it look better towards the end